charts/s1-agent/templates/_helpers.tpl

```diff
@@ -424,9 +424,15 @@ requests:
 {{- define "agentResources" -}}
 limits:
 {{ toYaml .Values.agent.resources.limits | indent 2 }}
+{{- if and .Values.configuration.platform.gke.autopilot (not (index .Values.agent.resources.limits "ephemeral-storage" | default 0)) }}
```

Comment:
I'm using `index` here since it's not allowed to do `.Values.agent.resources.limits.ephemeral-storage`. It's throwing
`Error: INSTALLATION FAILED: parse error at (s1-agent/templates/_helpers.tpl:433): bad character U+002D '-'`
(it doesn't like the `-`)